src/backend/vulkan/src/device.rs

```diff
@@ -713,6 +713,22 @@ impl d::Device<B> for Device {
         })
     }
 
+    fn create_texel_buffer_view(
+        &mut self, view: &n::BufferView, format: format::Format
+    ) -> Result<n::TexelBufferView, buffer::ViewError> {
+        Ok(n::TexelBufferView { raw: 
+            match self.raw.0.create_buffer_view(vk::BufferViewCreateInfo {
```

Comment:
hmm, where are the match arms?

Reply:
Whoops, that is not a match statement anymore.